css/properties/list-style-image.json

```
@@ -15,7 +15,8 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Prior to Firefox 86 this property did not accept an <image> type, and required the url of an image."
```

some style suggestions here

```suggestion
              "notes": "Before Firefox 86, this property did not accept an <code>&lt;image&gt;</code> type, and required the URL of an image."
```